nilearn/_utils/param_validation.py

```diff
@@ -5,7 +5,7 @@
 import numbers
 import warnings
 
-from .compat import _basestring
+from .compat import _basestring, izip
```

Comment:
You should really really really configure your editor to run pyflake8. This would have shown you that izip is unused.